notebooks/MAST/TESS/beginner_tess_exomast/beginner_tess_exomast.ipynb

```diff
@@ -0,0 +1,325 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"top_page\"></a>\n",
+    "# Exoplanet Data and TESS Light Curves Using Python Requests\n",
+    "\n",
+    "Exo.mast provides several API tools to obtain the planet parameter information for exoplanets available via that interface. Also, when available you can use these tools to download and plot the TESS light curves.\n",
+    "\n",
+    "It is all done via a RESTful API, but it is not available through astroquery. We will show you how to construct the url, do the ***get*** Request and manipulate the resulting information.\n",
+    "\n",
+    "[Imports](#imports)<br>\n",
+    "[Get Planet Names](#names)<br>\n",
+    "[Get Planet Properties](#props)<br>\n",
+    "[Get TCE Light Curve](#lightcurve)<br>\n",
+    "[Get TCE Phase Folded Plot](#phased)<br>\n",
+    "[Additional Resources](#additional)<br>\n",
+    "[About This Notebook](#about)<br>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "***"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"imports\"></a>\n",
+    "## Import statements\n",
+    "We need the following python packages to do the imports.\n",
+    "- **numpy** for array manipulation\n",
+    "- **requests** to make the RESTful API get requests\n",
+    "- **matplotlib.pyplot** to plot\n",
+    "- **pandas** to manipulate data\n",
+    "- **IPython.display.display** and **IPython.display.HTML** to render the plot delivered by the API\n",
+    "\n",
+    "We also specify two base URLs that we use repeatedly for these requests. The planet url is used to query the planet table.  The Data Validation url is used to query the Data Validation database which holds the header and light curves that are contained in the Data Validation time series files. There is one entry per TCE. TCEs are events (they have a period and an epoch) found in the 2-min cadence data by the TESS Ames Pipeline.  Targets with TCEs have this type of time series which have been flattened and can be easily folded at the period of the reported transit. "
```

Comment:
It might be useful to explain "TCE" more.  You could use the same definition Scott did in his Intro to DV data notebook: https://github.com/spacetelescope/notebooks/blob/master/notebooks/MAST/TESS/beginner_how_to_use_dvt/beginner_how_to_use_dvt.ipynb